### PR TITLE
Catch file conversion error and add a lot of invalid files.

### PR DESCRIPTION
**Pull request by DanBmh**
Note: I did not check these automatically excluded files. They threw an error in my deepspeech training. 
But earlier i did check some of the tuda files throwing the error and they all had a wrong transcription.

### CODE REVIEW

**Comment by ynop** [comment]
Thanks for your work.

A few comments:
- In the commit "Added files with less than one byte.", I checked one file and I could not confirm that is invalid (common_voice_de_18235793.mp3). I checked those, because such files should already be filtered. (I did it here: https://github.com/german-asr/megs/blob/master/scripts/validate.py)
- Could you elaborate a bit more on how you did the automatic check in the third commit?
  Predicting with a model trained on the same data?
- Don't forget to run flake8 to check code style (See Travis build)

**Comment by DanBmh** [comment]
I checked the file you mentioned again but in my dataset it has 0 bytes in the file explorer. Which common voice version do you use? I have this one: de_538h_2019-12-10

Do you run the validation check before or after the mp3->wav conversion. I did encounter an error  with a 0 byte file there which is why i did the second commit. After that i filtered out all 0 byte files.

I did train a new model using the pretrained weights from the English deep speech model. This did work very well with the voxforge dataset  but in the other datasets there are some files that result in an infinite loss. Those files i did remove and started the training again. The error does normally not occur in the first training step, but after some 1000 steps. Sometimes the training run more than one epoch before i got this error.

**Comment by ynop** [comment]
Ah, there is already a new one ...
I guess I have downloaded it in november
But it is a bit unfortunate, that the utterance changed.
Is it a completely different utterance or is just the file corrupt in the new version?
For "common_voice_de_18235793", is the transcription "Lass uns mal dort drüben bei der Touristeninformation nachfragen." as well in the new version?

I did the validation before converting to wav.

**Comment by DanBmh** [comment]
The transcription is the same

**Comment by ynop** [comment]
Ok, I checked a few more and at least it seems that the utterances are the same, but the files in the new version are somehow corrupt.
Therefore we can keep the previous list of invalid files.

Before I merge, could you:
- you merge commit b8927a2c4fdae039e2d017667e7588832281f609 and 0483ae8ecc0dfdb553c44f8d38699ff56d191df9
- add the description of the automatic file checks (from the comment above) into the commit description
